docs/sources/index.md

```diff
@@ -1,7 +1,6 @@
 # Event sources
 
-The Triggermesh event Sources are still under active development so this document will be evolving rapidly over the next
-months. Feel free to regularly check back for updates on current Targets as well as the addition of new ones!
+The following is a list of Triggermesh event Sources some available as open source projects some available as hosted solutions on our Cloud.
```

Comment:
Shall we use the same phrasing in the targets' index?

Reply:
yes